### PR TITLE
Use images in CourseCards

### PR DESCRIPTION
**Pull request by Erin007**
CourseCards now show an image with icons. 

BEFORE:
course cards without images

AFTER: 
course cards with images

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker @joshlory

File: apps/src/templates/teacherHomepage/CourseCard.jsx [hunk 1]
```diff
@@ -28,8 +26,8 @@ const styles = {
     paddingTop: 10,
     paddingBottom: 5,
     marginTop: 15,
-    fontSize: 18,
-    fontFamily: '"Gotham 3r", sans-serif',
```
Comment (breville) [inline apps/src/templates/teacherHomepage/CourseCard.jsx:32]: We should still have a fallback to sans-serif.

**Comment by Bjvanminnen** [comment]
Since this is hardcoded on the client, does that mean we no longer need to pass it down via server https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/user.rb#L903?

File: apps/src/templates/teacherHomepage/CourseCard.jsx [hunk 2]
```diff
@@ -130,23 +127,28 @@ const CourseCard = React.createClass({
   },
 
   render() {
-    const { name, description, link } = this.props;
+    const { name, description, link, image } = this.props;
+    const filenameToImgUrl = {
+      "large_teal_icons": require('../../../static/large_teal_icons.png'),
```
Comment (breville) [inline apps/src/templates/teacherHomepage/CourseCard.jsx]: worth making these absolute paths now?

**Comment by Erin007** [comment]
@Bjvanminnen Yes, I removed image since it doesn't need to be passed down.  In the future, if we decide each course has some unique image that goes with it, we can add image back in.

**Comment by Bjvanminnen** [comment]
You'll probably need to remove from `user_test.rb` as well

File: apps/src/templates/teacherHomepage/CourseCard.jsx (hunk 1 above)
Comment (Erin007) [reply apps/src/templates/teacherHomepage/CourseCard.jsx:32]: aha! We should remember that 3r won't work, but 4r will.

**Pull request merged**